app/scripts/factories/project.factory.js

```diff
@@ -47,6 +47,11 @@ function cultuurnetProjectFactory(IntegrationType, ProjectStatuses) {
       this.sapiVersion = jsonObject.sapiVersion || '2';
       this.contentFilter = jsonObject.contentFilter || '';
 
+      this.insightlyUrl = 'https://crm.na1.insightly.com/details/Project/' + this.insightlyProjectId;
+      if (jsonObject.opportunityIdInsightly && !jsonObject.projectIdInsightly) {
+        this.insightlyUrl = 'https://crm.na1.insightly.com/details/Opportunity/' + this.insightlyProjectId;
```

Comment:
Should this not use the `opportunityIdInsightly` instead at the end of the URL? Why do we use the `insightlyProjectId` if we have a `opportunityIdInsightly`?

Reply:
That is indeed more safe for the future. But for now the `insightlyProjectId` will be the `opportunityIdInsightly` if it's just an Opportunity inside Insightly.

Reply:
I made the code a bit more explicit and future proof.

Reply:
Thanks! Is it possible that there's an actual bug now? 😄 This links to an opportunity using the new project id if I'm not mistaken https://github.com/cultuurnet/projectaanvraag-angular/pull/59/files#diff-a476b2a743a63ecd3e017f601e9b25425825df080464fc66c33e15da93730f08R52

Reply:
Indeed, I made a copy/paste error here. Should be fine now 😅

Reply:
Thanks for the quick fix!